blog/20210808-mate-1-26-released.md

```diff
@@ -0,0 +1,450 @@
+
+## What's new in MATE 1.26
+
+The theme for this release has been adding new functionality to the MATE Desktop while maintaining the look and feel that we all know and love. While all the added features are surely quite exciting we also did not forget to do tons of bugfixing, modernising the code base and optimizing the performance.
+
+The headline changes in MATE Desktop 1.26 are:
+
+* We added [Wayland](https://wayland.freedesktop.org/) support for Atril, System Monitor, Pluma, Terminal and other components of the Desktop. Furthermore Calculator and Terminal can now be built with [Meson](https://mesonbuild.com/).
+* **Pluma** was subject to massive improvements.
+    * There is a **new mini map** which gives you instant overview over the content.
+    * The **new grid background pattern** turns Pluma into a writing pad.
+    * The sort plugin now supports undo actions.
+    * We added a show/hide line-numbers shortcut, namely <kbd>Ctrl</kbd> + <kbd>Y</kbd>.
+    * The preferences dialog needed a redesign to catch up with all the new features.
+* Together with the brand new **Pluma Plugins** our text editor **can be turned into a feature-rich IDE** with support for *Bracket completion*, *Comment/Uncomment code*, *Built-in terminal* and *Wordcompletion*.
+* The **Control Center** features an improved Windows preferences dialog with more options. In the Display dialog we added an option for display scaling.
+* Notifications became even more useful as they gained support for hyperlinks. And we finally **added a Do not Disturb applet**.
+* Speaking of applets, the Window List applet has a new option to disable mouse scrolling. The optional window thumbnails are now **crisp and clear as they are rendered as cairo surfaces**.
+* The Netspeed applet shows more information by default and gained [netlink](https://www.man7.org/linux/man-pages/man7/netlink.7.html) support.
+* The **Calculator** was ported to the [GNU MPFR](https://www.mpfr.org/)/[MPC](http://www.multiprecision.org/mpc) library which means more precision, faster computation and additional functions (e.g. Riemann zeta function, Gauss error function).
+    * We finally **added a history to show recent computations**.
+    * Calculator is now resizeable when the history is activated.
+    * Vastly improved integer factorization speed.
+    * Much faster modular arithmetic evaluation (modular exponentiation ability).
+* In the file manager **Caja** you can select **a new Bookmarks sidebar**. Caja gained **support for formatting drives** from the context menu. The **Caja Actions** extension which allows you to add arbitrary programs to be launched through the context menu is now part of the Desktop.
+* Scrolling through large documents in **Atril** is a lot faster now since a slow linear search has been replaced by a binary tree search. The document viewers **memory footprint got even smaller** as a EvWebView is now only created if needed.
```

Comment:
```suggestion
* Scrolling through large documents in **Atril** is a lot faster now since a slow linear search has been replaced by a binary tree search. The document viewer's **memory footprint got even smaller** as a EvWebView is now only created if needed.
```